library/data/geofoncier_wms_getFeatureInfo/README.md

```diff
@@ -0,0 +1,13 @@
+<h2>Permet de remplacer la réponse GetFeatureInfo de la couche WMS DOSSIERS_LOCALISANTS retournée par le WMS GeoFoncier</h2>
```

Comment:
```suggestion
# Remplacer la réponse GetFeatureInfo pour le WMS Geofoncier

French use-case only 🇫🇷

Permet de remplacer la réponse GetFeatureInfo de la couche WMS DOSSIERS_LOCALISANTS retournée par le WMS GeoFoncier
```

Let's remove HTML and use markdown insead

Reply:
Done :)